docs/conf.py

```diff
@@ -22,17 +21,21 @@
 import shutil
 import sys
 
-print(os.path.abspath('./shap'))
-sys.path.insert(0, os.path.abspath('..'))
+print(os.path.abspath("./shap"))
+sys.path.insert(0, os.path.abspath(".."))
 
 # make copy of notebooks in docs folder, as they must be here for sphinx to
 # pick them up properly.
-NOTEBOOKS_DIR = os.path.abspath('example_notebooks')
+NOTEBOOKS_DIR = os.path.abspath("example_notebooks")
 if os.path.exists(NOTEBOOKS_DIR):
     import warnings
-    warnings.warn('example_notebooks directory exists, replacing...')
+
+    warnings.warn("example_notebooks directory exists, replacing...")
     shutil.rmtree(NOTEBOOKS_DIR)
-shutil.copytree(os.path.abspath('../notebooks'), NOTEBOOKS_DIR, )
+shutil.copytree(
+    os.path.abspath("../notebooks"),
+    NOTEBOOKS_DIR,
+)
```

Comment:
A tiny minor point - this looks an instance of black's [magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma). This is useful when we anticipate that the list of arguments will grow or change frequently, but otherwise it's probably a little neater to remove that comma and re-run black to have it on one simple line.

I'll approve in any case, that's just a general comment in case we start running black more broadly

Reply:
Sure,I'll make it one line